src/js/schemas/job-schema/Schedule.js

```diff
@@ -35,7 +35,11 @@ const General = {
     },
     cron: {
       title: 'CRON Schedule',
-      description: 'Enter your schedule in CRON format, e.g. 0 20 * * *',
+      description: (
+        <span>
+          Enter your schedule in CRON format, e.g. <i>0 0 20 * * *</i>. <a href="https://dcos.io/docs/1.8/usage/jobs/getting-started" target="_blank">Read the docs</a> for more info.
```

Comment:
I would suggest a `<br/>` after the first description and putting the `<a>` in a `<small>` in order to differentiate style.